### ISSUE TITLE
Access token issues

### ISSUE THREAD

**Issue by ashceryth** [michaelherger/Spotty-Plugin#148]
Hello Michael,

I'm currently using version (v4.11.123) from your testing repo because it helped with issues I had playing podcasts (which are very important to me :)

However, starting today I'm facing issues with Spotty where it says that it's unable to obtain an access token. I already tried to reauthenticate, which seems to help until I want to play a podcast. This is what I see in the logs:

```
[24-11-05 18:48:19.7555] Plugins::Spotty::API::__ANON__ (1464) API call: shows/***
[24-11-05 18:48:19.7555] Plugins::Spotty::API::__ANON__ (1468) error: 429 Too Many Requests
[24-11-05 18:48:19.7557] Plugins::Spotty::API::error429 (1526) Zugriffsrate überschritten für: shows/***; nächster Versuch in 11742 Sekunden.
[24-11-05 18:48:28.6695] Plugins::Spotty::OPML::__ANON__ (189) Failed to get featured playlists and/or token - do not continue
[24-11-05 18:48:36.9867] Plugins::Spotty::OPML::__ANON__ (189) Failed to get featured playlists and/or token - do not continue
[24-11-05 18:48:40.0469] Plugins::Spotty::OPML::__ANON__ (189) Failed to get featured playlists and/or token - do not continue
```

There are also log entries for various podcasts like this:

```
[24-11-05 18:48:01.9514] Plugins::Spotty::OPML::podcastList (1360) This show needs inspection: [...]
```

I'm already using a Client ID.

Is there anything I could try to resolve the issue?

Thanks for all your great work and this awesome plugin!

**Comment by michaelherger** [comment]
429 means they are rejecting you because you sent them too many requests. Did you see more of them before that line? Because 10k seconds is pretty long: you‘d usually only have to wait a few minutes, not hours…

**Comment by ashceryth** [comment]
Yes, but I'm wondering what caused the "too many" requests in the first place. It seems to be related to podcasts. I was able to play a random playlist without any issues after the reauth but as soon as I tried to play a podcast it stopped working and I got the access token message again.

I did a few tests after seeing the access token message for the first time. So serveral reauths and I also tried to reinstall the plugin.
